import_logs.py

```diff
@@ -498,7 +502,8 @@ def get(self, key):
     'elb': RegexFormat('elb', _ELB_LOG_FORMAT, '%Y-%m-%dT%H:%M:%S'),
     'nginx_json': JsonFormat('nginx_json'),
     'ovh': RegexFormat('ovh', _OVH_FORMAT),
-    'haproxy': RegexFormat('haproxy', _HAPROXY_FORMAT, '%d/%b/%Y:%H:%M:%S.%f')
+    'haproxy': RegexFormat('haproxy', _HAPROXY_FORMAT, '%d/%b/%Y:%H:%M:%S.%f'),
+    'gandi': RegexFormat('gandi', _GANDI_FORMAT, '%d/%b/%Y:%H:%M:%S')
```

Comment:
This needs to match the name as above: `_GANDI_SIMPLE_HOSTING_FORMAT `

Reply:
I just corrected this error and added the tests for the "Gandi" format